website/versioned_docs/version-v1.2/cli.md

```diff
@@ -239,27 +239,6 @@ Example output:
 [✓] Executed mergeStateAq(); gas used: 1004720
 ```
 
-### Coordinator: merge message tree
```

Comment:
Please remove all changes to the documentation, 1.2 is a released version and will not change, 2.0 is about to be released in a couple of weeks and this grant work will be part of 3.0 so in case of any documentation changes let's create a new folder inside the doc website

Reply:
fixed